src/Credentials/AppIdentityCredentials.php

```diff
@@ -104,11 +112,27 @@ public function fetchAuthToken(callable $httpHandler = null)
     $scope = is_array($this->scope) ? $this->scope : explode(' ', $this->scope);
 
     $token = AppIdentityService::getAccessToken($scope);
+    $this->lastReceivedToken = $token;
 
     return $token;
   }
 
   /**
+   * Implements FetchAuthTokenInterface#getLastReceivedToken.
+   */
+  public function getLastReceivedToken()
+  {
+    if ($this->lastReceivedToken) {
+      return [
+        'access_token' => $this->lastReceivedToken['access_token'],
+        'expires' => $this->lastReceivedToken['expiration_time'],
```

Comment:
Yeah, I agree. I thought the [oauth spec](https://tools.ietf.org/html/rfc6749) used "expires", but I can find no examples of this.